packages/fluent-theme/src/components/activity/private/useActivityAuthor.ts

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react';
+import { getOrgSchemaMessage, type WebChatActivity } from 'botframework-webchat-core';
+
+export default function useActivityAuthor(activity?: WebChatActivity | undefined) {
+  return useMemo(() => {
```

Comment:
Maybe typing it here so we are sure the structure in line 8-13 is a `Person`. 

```ts
  return useMemo<Person>(() => {
```

Reply:
The type is not shared, so we don't rely on it.